Configuration/Settings.yaml

```diff
@@ -1,3 +1,24 @@
+Flowpack:
+  ElasticSearch:
+    ContentRepositoryAdaptor:
+      driver:
+        version: '2.x'
```

Comment:
I would just call the versions "1" and "2", same as in the Namespace (Version1 / Version2)